src/Point/Point.cs

```diff
@@ -292,29 +306,44 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
         private void HandlePreventedPointPosition(GenericPointPosition commandedPosition, ref GenericPointPosition pointPosition, ref GenericDegradedPointPosition degradedPointPosition)
         {
             pointPosition = commandedPosition;
-            switch (_simulatedPosition.PreventedPosition)
+            switch (_simulatedPointState.PreventedPosition)
             {
                 case PreventedPosition.PreventedLeft:
                     if (commandedPosition == GenericPointPosition.Left)
                     {
-                        degradedPointPosition = RespectAllPointMachinesCrucial(_simulatedPosition.DegradedPointPosition);
+                        degradedPointPosition = RespectAllPointMachinesCrucial(_simulatedPointState.DegradedPointPosition);
                         pointPosition = GenericPointPosition.NoEndPosition;
                     }
                     break;
                 case PreventedPosition.PreventedRight:
                     if (commandedPosition == GenericPointPosition.Right)
                     {
-                        degradedPointPosition = RespectAllPointMachinesCrucial(_simulatedPosition.DegradedPointPosition);
+                        degradedPointPosition = RespectAllPointMachinesCrucial(_simulatedPointState.DegradedPointPosition);
                         pointPosition = GenericPointPosition.NoEndPosition;
                     }
                     break;
                 case PreventedPosition.PreventTrailed:
-                    degradedPointPosition = RespectAllPointMachinesCrucial(_simulatedPosition.DegradedPointPosition);
+                    degradedPointPosition = RespectAllPointMachinesCrucial(_simulatedPointState.DegradedPointPosition);
                     pointPosition = GenericPointPosition.UnintendedPosition;
                     break;
             }
-            _simulatedPosition.PreventedPosition = PreventedPosition.None;
-            _simulatedPosition.DegradedPointPosition = GenericDegradedPointPosition.NotDegraded;
+            _simulatedPointState.PreventedPosition = PreventedPosition.None;
+            _simulatedPointState.DegradedPointPosition = GenericDegradedPointPosition.NotDegraded;
+        }
+
+        private void HandleAbilityToMove(ref GenericPointPosition pointPosition, ref GenericDegradedPointPosition degradedPointPosition)
+        {
+            switch (_simulatedPointState.AbilityToMove)
+            {
+                case AbilityToMove.AbleToMove:
+                    break;
+                case AbilityToMove.UnableToMove:
+                case AbilityToMove.Undefined:
+                    _logger.LogWarning("Point is unable to move.");

```

Comment:
is it actually? What happens when the point receives a move point command and pretends to be unable to move?

Reply:
Spec for this: Eu.P.5371